content/Software Support/Upload/If-your-board-is-not-detected-by-Arduino-IDE.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "If your board is not detected by Arduino IDE"
+id: 4412955149586
+---
+
+Learn what to do if your Arduino board is missing from the board selector or from the _Tools > Port_ menu in Arduino IDE.
+
+> [!NOTE]
+> If you're using Arduino Cloud Editor, see [If your board is not detected by Arduino Cloud Editor](https://support.arduino.cc/hc/en-us/articles/360018131160-If-your-board-is-not-detected-by-Arduino-Cloud-Editor).
+
+## Troubleshooting
```

Comment:
I would recommend including this troubleshooting step:

- Disconnect and reconnect your board

Motivation:

- This is a simple and quick action that will reliably power cycle the board. There are situations where the board may not respond to the reset button, or "double press" bootloader reset. An example of this is the UNO R4 WiFi, where the ESP32 is set to Download mode.
- It can also help by triggering system prompts to allow the USB device to connect, if the user previously failed to do so:
   ![image](https://github.com/user-attachments/assets/0d2f6ae4-b977-4bf2-828e-82ec0c3fe964)